### DISCUSSION TITLE
2주차 미팅

### DISCUSSION THREAD

**Discussion by sejoonkimmm** [GunGonGamLee/ft_transcendence#22] [General]
# 의논할 것
- Wireframe 논의
- 게임 서버와 백 + 프론트 서버를 나눌 것인가?
- 나눈다면, 게임 서버의 DB에는 Redis를 도입할 것인가?

# 해야할 것
- 디자인 70%
- swagger 작성
- ELK TLS 통신 구현하기
- nginx에 방화벽 구성 / modsecurity 구성
- hascorp vault 컨테이너 작성 / .env, API 키, TLS 인증서 보관 고려해보기

**Comment by nyj001012** [comment]
디자인 적어도 사용자 지정 게임 방 내부, 토너먼트 & 1 vs 1 화면 정도는 이번주엔 마쳐야겠네요
